code/script.js

```diff
@@ -1,47 +1,105 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
+const docBotForm = document.getElementById('doc-bot-form')
+const userInput = document.getElementById('user-input')
+let name = " "
 
 // Global variables, if you need any, declared here
+let questionCounter = 0
 
-// Functions declared here
+// Functions declared here.
+const userReply = () => {
+    showMessage(userInput.value, 'user')
+}
+const botReply = (message) => {
+    showMessage(message, 'bot')
+}
```

Comment:
you could add setTimeout here, what would make your code much cleaner and shorter 😸

Reply:
I don't quite understand?  Can you clarify a bit more?

Reply:
``` 
const botReply = (message, time) => {
    setTimeout(showMessage(message, 'bot'), time)
}
```
This would save you a lot o typing...

Reply:
Ah OK! we fixed that in the new push